configs/solo/solo_r50_fpn_1x_coco.py

```diff
@@ -0,0 +1,14 @@
+_base_ = [
+    '../_base_/datasets/coco_instance.py', '../_base_/models/solo_r50_fpn.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+# optimizer
+optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001)
+optimizer_config = dict(grad_clip=dict(max_norm=35, norm_type=2))
+# learning policy
+lr_config = dict(
+    policy='step',
+    warmup='linear',
+    warmup_iters=500,
+    warmup_ratio=1.0 / 3,
+    step=[9, 11])
```

Comment:
8, 11 actually achieves similar performance, we should use our default config if [9,11] is unnecessary.